### ISSUE TITLE
[QA] Linux: saved credentials do not work when updating from 2.8.2 to 2.9.0-beta1

### ISSUE THREAD

**Issue by jnweiger** [owncloud/client#8902]
Seen with client 2.9.0-beta1 (build 4915) on Mint Linux 20.2 (Uma) -- based on Ubuntu 20.04

* Install client 2.8.2 (build 4246), connect to several servers. the credentials get stored in the desktop's password manager seahorse.
* Quit the 2.8.2-client, restart the 2.8.2-client. The password manager delivers the login credentials, all servers automatically reconnect.

![image](https://user-images.githubusercontent.com/1108546/129644048-7afaaa5c-9e04-4655-9f32-8b37c66c0ced.png)

----

* Upgrade to client-2.9.0-beta1
* The oauth2 serves cloud.damken.com and cloud.owncloud.com cause browser login roundtrips with the servers.
* The basic auth server opens a password prompt.

![image](https://user-images.githubusercontent.com/1108546/129644735-914cac56-d518-481f-a8f5-257d0fe52342.png)

----

* Downgrade back to 2.8.2
* All servers immediately reconnect without browser roundtrips  or password requestors.

**Comment by jnweiger** [comment]
On windows 10_20H2v2 stored credentials are correctly re-used after the upgrade.

**Comment by TheOneRing** [comment]
Expected behaviour is that 2.8 credentials are migrated to 2.9.

Log files or it didn't happen.

**Comment by jnweiger** [comment]
[290b1-restart-log.zip](https://github.com/owncloud/client/files/6998764/290b1-restart-log.zip)
[282-restart-log.zip](https://github.com/owncloud/client/files/6998765/282-restart-log.zip)

**Comment by TheOneRing** [comment]
It  was a deployment issue, libsecret was not part of the appimage, therefore no keychain access succeeded.
The client also prompted for a pw /oath on every start.

**Comment by jnweiger** [comment]
Retested with owncloud-client 2.9.0 RC1 on Fedora 33

* install client 2.8.2
* connect to demo.owncloud.org, what for sync to complete
* quit the client
* upgrade to 2.9.2-rc1
* start, the client asks for password

![image](https://user-images.githubusercontent.com/1108546/131467984-d0366f85-1bfb-43c3-bd91-6d5f52e0a707.png)

**Comment by TheOneRing** [comment]
@jnweiger please consider closing issues if you find them fixed.

**Comment by TheOneRing** [comment]
sometimes your reports are hard to parse...

Does it only happen when migration from official repo to app image? 
We don't support migration from gnome keyring to libsecret so something might get lost there.

It worked on my system and before the resent fix credentials didn't work at all with the app image.

Please provide logs for the upgrade issue.
